openforcefield/topology/molecule.py

```diff
@@ -669,16 +774,17 @@ def molecule_virtual_site_index(self):
         #       Deleting atoms/molecules in the Topology would have to invalidate the cached index.
         return self._molecule.virtual_sites.index(self)
 
-    @property
-    def molecule_particle_index(self):
-        """
-        The index of this VirtualSite within the the list of particles in the parent ``Molecule``.
-        Note that this can be different from ``molecule_virtual_site_index``.
+    # @property
+    # def molecule_particle_index(self):
```

Comment:
Please either remove this or make it a `NotImplementedError` stub; it looks un-used now but I'm not sure if that will always be the case